src/main/java/Aples.java

```diff
@@ -0,0 +1,98 @@
+import org.w3c.dom.ls.LSInput;
```

Comment:
May I know what is this imported library for? Not sure where it's used hmm...
```suggestion
import org.w3c.dom.ls.LSInput;
```